tasks/tests/test_view_single_task.py

```diff
@@ -17,14 +17,44 @@ def test_with_employee_logged_in(self, client, task_1, employee_1):
         assert response.status_code == 200
         assert response.context['task'] == task_1
 
-    def test_view_task_form(self, client, task_1, employee_1):
+    @pytest.mark.parametrize('form, form_type',
+                             [
+                                 ('task_form', ViewTaskForm),
+                                 ('comment_form', CommentForm),
+                             ],
+                             ids=[
+                                 "task_form",
+                                 "comment_form",
+                             ])
+    def test_view_form(self, client, task_1, employee_1, form, form_type):
         client.login(username=employee_1.user.username, password='password')
         response = client.get(f'/tasks/{task_1.id}')
-        assert isinstance(response.context.get('form'), ViewTaskForm)
+        assert isinstance(response.context.get(form), form_type)
 
     def test_change_status(self, client, task_1, employee_1):
         client.login(username=employee_1.user.username, password='password')
         assert task_1.status == Status.IN_PROGRESS
-        client.post(f'/tasks/{task_1.id}', data={'status': 'DONE'})
+        client.post(f'/tasks/{task_1.id}', data={'taskSubmit': True, 'status': 'DONE'})
         updated_task = Task.objects.get(id=task_1.id)
         assert updated_task.status == Status.DONE
+
+    @pytest.mark.parametrize('content, difference',
+                             [
+                                ('This is a test comment', 1),
```

Comment:
Great use of parametrize!!!
maybe you can add a few more tests like:
 - invalid too long comment
 - valid long comment
 - valid with special characters

Reply:
Done.
Note: I considered 'special characters' to be the set of characters that is defined as python's string.punctuation.